src/Translatable/Translatable.php

```diff
@@ -162,6 +162,10 @@ public function getAttribute($key)
                 return;
             }
 
+            if ($this->hasGetMutator($key)) {
+                return $this->mutateAttribute($key, $this->getTranslation($locale)->$key);
```

Comment:
Thank you for the reply, @dimsav .

`$this->getAttributeValue($key)` will check the Model's `$attribute` array. and does not get the value from the translation. then internally checks if the `$key` has a mutator, then calls the mutator with the Model's `$value` (which is not translated.).

So calling `$this->getAttributeValue($key)` does not mutate translated attributes. 

But, you are right, using the `$this->getAttributeValue($key)` makes more sense, so I will change the code (by checking if `$key` has a mutator, then push it to `$attributes` before calling `$this->getAttributeValue($key)`.